packages/_shared/src/ReleaseEntityStatusBadge/ReleaseEntityStatusBadge.tsx

```diff
@@ -34,14 +35,19 @@ const config: Record<ReleaseEntityStatus, { label: string; variant: BadgeVariant
 
 export function ReleaseEntityStatusBadge({ className, status }: ReleaseEntityActionBadgeProps) {
   const badgeConfig = config[status];
+  const statusLabel = badgeConfig.label;
```

Comment:
you have to change it in the config:
the config would need to be a function to generate it
inside you can then use `t` to build the messages with static messages
in the componetn here yo uwould say something like this:
const badeConfig = getConfig(status) // getConfig()[status]

Reply:
yes exactly